package.json

```diff
@@ -12,27 +12,37 @@
     "generate:pagopa-api-walletv2": "rimraf generated/definitions/pagopa/walletv2 && mkdir -p generated/definitions/pagopa/walletv2  && gen-api-models --api-spec $npm_package_pagopa_api_walletv2 --out-dir ./generated/definitions/pagopa/walletv2 --no-strict --request-types --response-decoders",
     "lint": "eslint . -c .eslintrc.js --ext .ts,.tsx",
     "lint-autofix": "eslint . -c .eslintrc.js --ext .ts,.tsx --fix",
-    "test": "jest --detectOpenHandles",
-    "test:coverage": "jest --detectOpenHandles --coverage",
+    "pretest:integration": "yarn build:test-cases",
+    "pretest:coverage-int": "yarn build:test-cases",
+    "test:unit": "jest -c jest.config.unit.js --detectOpenHandles",
+    "test:integration": "jest -c jest.config.integration.js --detectOpenHandles --runInBand",
+    "test:coverage-unit": "jest -c jest.config.unit.js --detectOpenHandles --coverage",
+    "test:coverage-int": "jest -c jest.config.integration.js --detectOpenHandles --runInBand --coverage",
+    "test": "yarn test:unit && yarn test:integration",
+    "test:coverage": "yarn test:coverage-unit && yarn test:coverage-int",
     "build": "parcel build src/index.ts src/*.pug",
+    "build:test-cases": "parcel build src/__integrations__/testCases/*.ts --out-dir testCases",
```

Comment:
the PR description says `yarn build:testCases` but here is `build:test-cases `

Reply:
Thanks, I've fixed

Reply:
@infantesimone I have fixed test names with the last commit. Now they should adhere to our standards